internal/cmd/branch/create.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/pkg/browser"
 	"github.com/planetscale/cli/internal/cmdutil"
 	"github.com/planetscale/cli/internal/printer"
+	"github.com/planetscale/planetscale-go/planetscale"
```

Comment:
Isn't this already being imported in the line below aliased as `ps` ?

Reply:
Good catch, thanks! Fixed in https://github.com/planetscale/cli/pull/306/commits/afe9e35433d292431d3669cfebd0c1a2bcc61636.